tests/test-sources/plugins/by-name/telekasten/default.nix

```diff
@@ -0,0 +1,10 @@
+{
+  example = {
+    plugins.snipe-nvim = {
```

Comment:
```suggestion
    plugins.telekasten = {
```

Reply:
Oops, was thinking of different plugin. It's now fixed.